### ISSUE TITLE
Verify that morning messages are being sent to participant 201

### ISSUE THREAD

**Issue by nickdotreid** [kpwhri/heartsteps#245]
**From Rebecca**
Not receiving morning survey notifications, notifications are disappearing after an hour. 

This is effecting participant 201.

We want to verify that this participant is receiving morning message notifications and their account is configured correctly.

**Comment by rahul-dcosta** [comment]
**More from Rebecca**

Participants affected: 102, 114, 201, 231
Information: So a quick update, what I have heard from all these participants is that they are not receiving notifications to complete their morning check-ins. The Morning check-in banner will still appear on the HeartSteps app when they open it, but they'll forget to do the morning survey because they aren't getting those reminders. I've reached out to 201 for further details about the notifications disappearing. Once I hear from 201, I'll let you know. I just wanted to give you an update because it looks like more people are experiencing similar issues.

Also what time should people receiving the notification? I originally thought that they should be receiving it at 8AM, but the time stamp on the notification is showing 6AM.

**Comment by nickdotreid** [comment]
The morning messages are [supposed to be sent at 6AM](https://github.com/kpwhri/heartsteps/blob/master/server/morning_messages/models.py#L44-L45) -- the intervention is completely separate from the walking suggestion times and NLM planning prompts (which I think start at 8am by default).

Helpful?

**Comment by rahul-dcosta** [comment]
Yep, I've informed Rebecca about this. I've also looked into the
notifications issue for the four affected participants. The dashboard
indicates that morning survey notifications send as expected for three of
the participants. The remaining participant (participant 114) does not
receive morning survey notifications but receives other types of
notifications (activity-suggestion, weekly-reflection, and activity-survey).

**Comment by rbraganca** [comment]
Hello you guys! I was wondering if there were any updates about fixing this issue for 114 who is still not receiving morning survey notifications. Just let me know thank you!
